src/app/audiologist-summary/audiologist-summary.component.ts

```diff
@@ -0,0 +1,231 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Is this new logic or is it copied from somewhere? Eg. what's the basis for the score formula?

Reply:
https://drive.google.com/open?id=16zz9ZFEVUVJK1hjLBFy7yzBu3o54wGsP